Documentation/ceph-upgrade.md

```diff
@@ -356,60 +275,41 @@ These images are tagged in a few ways:
 * Ceph major version tags (e.g., `v13`) are useful for development and test clusters so that the
   latest version of Ceph is always available.
 
-**Ceph containers other than the official images above will not be supported.**
+**Ceph containers other than the official images from the registry above will not be supported.**
 
-### Example upgrade to Ceph Mimic
+### Example upgrade to Ceph Nautilus
 
 #### 1. Update the main Ceph daemons
 The majority of the upgrade will be handled by the Rook operator. Begin the upgrade by changing the
 Ceph image field in the cluster CRD (`spec:cephVersion:image`).
+<!--                  TODO: CEPH IMAGE BELOW NEEDS UPDATE -->
```

Comment:
We can remove the TODO, right? The ceph image is always in flux and we just didn't get 14.2.1 in time for the rook release.